components/org.wso2.carbon.identity.oauth/src/test/java/org/wso2/carbon/identity/oauth2/OAuth2ScopeServiceTest.java

```diff
@@ -258,12 +258,16 @@ private void insertAppId(String uuid) throws Exception {
 
         try (Connection connection = IdentityDatabaseUtil.getDBConnection()) {
             String sql = "INSERT INTO SP_APP (TENANT_ID, APP_NAME, UUID) VALUES (?,?,?)";
-            PreparedStatement ps = connection.prepareStatement(sql);
-            ps.setInt(1, 1);
-            ps.setString(2, "dummyAppName");
-            ps.setString(3, uuid);
-            ps.execute();
-            IdentityDatabaseUtil.commitTransaction(connection);
+            try (PreparedStatement ps = connection.prepareStatement(sql)) {
```

Comment:
You can add all close-able resources to the same `try-with-resources` statement to prevent nesting and boilerplate code. Refer https://www.baeldung.com/java-try-with-resources#resources

Reply:
Resolved